sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -108,6 +109,7 @@
     private boolean multipleWriteRegionsEnabled = true;
     private boolean readRequestsFallbackEnabled = true;
     private boolean clientTelemetryEnabled = false;
+    private ApiType apiType = ApiType.SQL;
```

Comment:
Please confirm that .NET sdk also using this default api type sql header for all their requests